ci-operator/step-registry/baremetalds/sno/rename/baremetalds-sno-rename-commands.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+cat >"${SHARED_DIR}"/recert-rename-sno.sh << "EOF"
```

Comment:
recert-rename-sno and rename_script are rather vague/similar names, it's hard to tell what their purpose is

Reply:
How about [this](https://github.com/openshift/release/pull/44454/files#diff-a925b502a3b0c057a42c58f40252cab6b1bf3a04a09c2ba9304141150aeac406R11) and [this](https://github.com/openshift/release/pull/44454/files#diff-a925b502a3b0c057a42c58f40252cab6b1bf3a04a09c2ba9304141150aeac406R31) instead?

Reply:
Oh only now I realized they're nested within each other. I thought they were at the same level which made the similar names confusing. But if they're nested it makes more sense, new names are good too